DEVDOCS-6375 - New GraphQL response headers (#948)
bigcommerce/docs: docs/start/about/index.mdx

{/* | `X-Request-ID` |  |  |  | */}

#### GraphQL-specific response headers

| Header | Possible Values | Description | Example |
|:-------|:----------------|:------------|:--------|

> | `X-Bc-Graphql-Complexity` | integer | Shows the complexity of the response content. If this exceeds `10000`, an error is returned. | `5241` |

Reviewer: ```suggestion
| `x-bc-graphql-complexity` | integer | Indicates the computed complexity score of the executed GraphQL query. If this exceeds `10000`, an error is returned. | `5241` |
``` 

The idea is that response has nothing to do with complexity, its calculated based on query (based on requested data)

> | `x-bc-gql-query-hash` | string | A hash of the query body, not including variables. This allows identification of specific queries among those with non-unique names. | `bd2e22a00004abd3ec9437a6f895` |

Author: ```suggestion
| `x-bc-gql-query-hash` | A `sha256` hash | A hash of the query body, not including variables. This allows identification of specific queries among those with non-unique names. | `bd2e22a00004abd3ec9437a6f895` |
```

> | `x-bc-graphql-query-hash` | A `sha256` hash | A hash of the query body, not including variables. This allows identification of specific queries among those with non-unique names. | `bd2e22a00004abd3ec9437a6f895` |

Reviewer: ```suggestion
| `x-bc-graphql-query-hash` | string | A 'sha256' hash of the query body, not including variables. This allows identification of specific queries among those with non-unique names. | `bd2e22a00004abd3ec9437a6f895` |
```

> | `x-bc-graphql-variables-hash` | string | A hash of the variables included in a query. | `44136fa355b3678a1146ad16f7e8` |

Author: ```suggestion
| `x-bc-graphql-variables-hash` | A `sha256` hash | A hash of the variables included in a query. | `44136fa355b3678a1146ad16f7e8` |
```

> | `x-bc-graphql-variables-hash` | A `sha256` hash | A hash of the variables included in a query. | `44136fa355b3678a1146ad16f7e8` |

Reviewer: ```suggestion
| `x-bc-graphql-variables-hash` | string | A `sha256` hash of the variables included in a query. | `44136fa355b3678a1146ad16f7e8` |
```

> | `x-bc-gql-operation-type` | One of `query`, `mutation`, or `subscription` | The operation type of the request. Currently, `subscription` is not supported. | `query` |

Author: ```suggestion
| `x-bc-graphql-operation-type` | One of `query`, `mutation`, or `subscription` | The operation type of the request. Currently, `subscription` is not supported. | `query` |
```

> | `x-bc-gql-operation-name` | string | The response name. This will return an empty string for anonymous requests. | `HomePageQuery` |

Author: ```suggestion
| `x-bc-graphql-operation-name` | string | The response name. This will return an empty string for anonymous requests. | `HomePageQuery` |
```

> | `x-bc-graphql-operation-name` | string | The response name. This will return an empty string for anonymous requests. | `HomePageQuery` |

Reviewer: Is it nullable, cc: @bc-tayvs ?

Reviewer 2: `The operation name within a query. Used with queries with multiple operations. This will return an empty string for anonymous requests`

Author: ```suggestion
| `x-bc-graphql-operation-name` | string |The operation name within a query. Used with queries with multiple operations. This will return an empty string for anonymous requests. | `HomePageQuery` |
```
